github/resource_github_team.go

```diff
@@ -95,8 +107,26 @@ func resourceGithubTeamCreate(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	if parentTeamIdString, ok := d.GetOk("parent_team_id"); ok {
+		/*
+			When creating nested teams via Terraform by looping through a module or resource
+			the parent team might not have been created yet (in "terraform apply" parallel runs),
```

Comment:
So there is no graph-like dependency map to traverse?

Reply:
> So there is no graph-like dependency map to traverse?

@danvaida 
It could be that another parallel Terraform  operation is responsible for creating the parent, but has slightly started later or its API call took slightly longer.